charts/azure-managed-cluster/Chart.yaml

```
@@ -0,0 +1,9 @@
+apiVersion: v2
+name: azure-managed-cluster
+description: A Helm chart to deploy an AKS cluster via the cluster-api-provider-azure ManagedCluster
+type: application
+maintainers:
+- email: jt572@cornell.edu
+  name: Jonathan Tong
```

```suggestion
  name: JonT828
```
As I mentioned [elsewhere](https://github.com/mboersma/cluster-api-charts/pull/3#issuecomment-2098980928), `helm-lint` is strict here and wants a GitHub handle.

---

It seems odd to me that the `name` field isn't actually the name. I think it makes more sense than the GitHub handle as you already have the email. Can we change the linter settings?

---

We could shut off the `maintainers` check but we decided not to in the conversation I linked to above.